apps/src/lib/kits/maker/util/boardUtils.js

```diff
@@ -0,0 +1,26 @@
+import {
+  ADAFRUIT_VID,
+  CIRCUIT_PLAYGROUND_EXPRESS_PID,
+  CIRCUIT_PLAYGROUND_PID
+} from '../portScanning';
+import {BOARD_TYPE} from '../boards/circuitPlayground/CircuitPlaygroundBoard';
+
+/**
+ * Detects the type of board plugged into the serial port. Defaults to BOARD_TYPE.OTHER.
+ */
+export function detectBoardTypeFromPort(port) {
```

Comment:
can we add unit tests for this function? it's pretty simple now, but my assumption is that we'll be refactoring this method to add logic for detecting micro:bit, and it'll be useful to have test coverage once we add more logic here

Reply:
Good call - added!